Bikeini/screens/Gamification.js

```diff
@@ -0,0 +1,114 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import {
+  StyleSheet, Text, View, ScrollView, Image,
+} from 'react-native';
+import headerStyle from './header';
+
+const profilePic = require('../assets/images/biker.png');
+
+
+const styles = StyleSheet.create({
+  background: {
+    backgroundColor: '#fff',
+  },
+  container: {
+    flex: 1,
+    alignItems: 'flex-start',
+    justifyContent: 'flex-start',
+  },
+  profile: {
+    height: 100,
+    width: 100,
+    margin: 10,
+  },
+  rowContainer: {
+    flexDirection: 'row',
+    backgroundColor: '#fff',
+  },
+  columnContainer: {
+    flexDirection: 'column',
+    backgroundColor: '#fff',
+    margin: 20,
+  },
+  categories: {
+    fontSize: 24,
+    margin: 10,
+  },
+  UserInfo: {
+    fontSize: 18,
+  },
+});
+
+class Gamification extends React.Component {
+    static navigationOptions = {
+      ...headerStyle,
+    };
+
+    constructor(props) {
+      super(props);
+      this.state = {
+      };
+    }
```

Comment:
Constructor not necessary but no need for removing it as it might be needed later on anyways 🐈